pkg/hs/urbit-king/lib/Urbit/Vere/Pier.hs

```diff
@@ -494,15 +494,15 @@ router slog waitFx Drivers {..} = do
 
 logEvent :: HasLogFunc e => Ev -> RIO e ()
 logEvent ev = do
-  logTrace $ "<- " <> display (summarizeEvent ev)
```

Comment:
I thought `logTrace` was supposed to be spammy than `logInfo`. But apparently not. It looks since like logTrace is a LevelOther, it's more important than errors. So yeah.

Reply:
With this at LogOther "trace", there's actually no way to turn it off. Note that logInfo is almost unused in the king, and it sits between debug and warn. Since the full event printing (next line) is at debug, this is a good level for this printout.